### PR TITLE
Issue #25 help system

### PR DESCRIPTION
**Pull request by plaursen**
Introducing an on-page help system / guided tour of page features.

First use: training log page.

### CODE REVIEW

File: FlightJournal.Web/Translations/LocalizedWebViewPage.cs [hunk 1]
```diff
@@ -16,7 +16,14 @@ public abstract class LocalizedWebViewPage<TModel> : WebViewPage<TModel>
         /// <example>@_("Country")</example>
         public string _(string en)
         {
-            return Internationalization.GetText(en, Internationalization.LanguageCode);
+            var s = Internationalization.GetText(en, Internationalization.LanguageCode);
+            if (s != " ")
+                return s.Trim();
+            return s;
+        }
+        public string __(string en)
```
Comment (janhebnes) [inline FlightJournal.Web/Translations/LocalizedWebViewPage.cs:24]: - [does this get picked up by po parser]

File: FlightJournal.Web/FlightJournal.Web.csproj [hunk 2]
```diff
@@ -138,8 +145,7 @@
     <Reference Include="System.Web.Entity" />
     <Reference Include="System.Web.ApplicationServices" />
     <Reference Include="System.ComponentModel.DataAnnotations" />
-    <Reference Include="System.Core" />
```
Comment (janhebnes) [inline FlightJournal.Web/FlightJournal.Web.csproj:141]: - [ ] Was System.Core not used... ?

File: FlightJournal.Web/Translations/LocalizedWebViewPage.cs (hunk 1 above)
Comment (janhebnes) [reply FlightJournal.Web/Translations/LocalizedWebViewPage.cs:24]: It does, nice work. we should consider comments tho

Comment (janhebnes) [inline FlightJournal.Web/Translations/LocalizedWebViewPage.cs:24]: ``
```suggestion
        /// <summary>
        /// Localized Translation (i18n) wrapped in ping for use on razor attributes
        /// </summary>
        /// <param name="en"></param>
        /// <returns></returns>
        /// <example><div class="col-xs-12" data-intro=@Html.Raw(__("Country") /></example>
        public string __(string en)
```

**Comment by janhebnes** [comment]
i all good with this change, feel free to merge when you see fit.

File: FlightJournal.Web/FlightJournal.Web.csproj (hunk 2 above)
Comment (plaursen) [reply FlightJournal.Web/FlightJournal.Web.csproj:141]: Hmmm. Guess not. Don't remember removing this explicitly. Might have been due to the update to .Net Framework 4.7.2

File: FlightJournal.Web/Translations/LocalizedWebViewPage.cs (hunk 1 above)
Comment (plaursen) [reply FlightJournal.Web/Translations/LocalizedWebViewPage.cs:30]: That actually took more or less a full day to get right, having considered all sorts of script-based hacks to get usage in strings to work.

The simplest would have been to grep for "_(" and let xgettext/po work on the output, but that would drop all real file references (and windows doesn't have grep...)

**Pull request merged**